.github/workflows/beam_PreCommit_SQL_Java11.yml

```diff
@@ -110,4 +110,10 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: SpotBugs Results
+          path: '**/build/reports/spotbugs/*.html'
+      - name: Publish SpotBugs Results
+        uses: jwgmeligmeyling/spotbugs-github-action@v1.2
+        if: always()
```

Comment:
Could you please explain why there's an 'if: always()' here?

Reply:
The step will be executed even if the previous steps failed.